github/Workflow.py

```diff
@@ -162,10 +170,13 @@ def create_dispatch(
             ref = ref.name
         if inputs is NotSet:
             inputs = {}
-        status, _, _ = self._requester.requestJson(
-            "POST", f"{self.url}/dispatches", input={"ref": ref, "inputs": inputs}
-        )
-        return status == 204
+        try:
+            self._requester.requestJsonAndCheck("POST", f"{self.url}/dispatches", input={"ref": ref, "inputs": inputs})
+        except GithubException.GithubException:
+            if throw:
+                raise
+            return False
+        return True
```

Comment:
Let's simplify the requester calls:
```suggestion
        url = f"{self.url}/dispatches"
        input = {"ref": ref, "inputs": inputs}

        if throw:
            self._requester.requestJsonAndCheck("POST", url, input=input)
            return True
        else:
            status, _, _ = self._requester.requestJson("POST", url, input=input)
            return status == 204
```